pkg/agent/datamodel/types.go

```diff
@@ -1749,6 +1748,8 @@ type CSEStatus struct {
 	CSEStartTime string `json:"cseStartTime,omitempty"`
 	// GuestAgentStartTime indicate starttime of GuestAgent, output from systemctl show walinuxagent.service -p ExecMainStartTimestamp
 	GuestAgentStartTime string `json:"guestAgentStartTime,omitempty"`
+	// Data contains key-value pairs passed out from CSE.
+	Data map[string]string `json:"data,omitempty"`
```

Comment:
Any reason to use a map instead of struct fields?

Reply:
I feel too much property is added, I am planning to move the other timestamp into this map as well.

Reply:
What do you think about using a nested struct? Something like `CSEStatusStats` or `CSEStatusMetadata`? I'd prefer to use a structured type to document which values can be expected.

Reply:
updated to bootDatapoints